### DISCUSSION TITLE
2024.5.27.0

### DISCUSSION THREAD

**Discussion by BornToBeRoot** [BornToBeRoot/NETworkManager#2729] [Releases]
[:page_facing_up: Changelog](https://borntoberoot.net/NETworkManager/docs/changelog/2024-5-27-0)
[:book: Documentation](https://borntoberoot.net/NETworkManager/docs/category/application)

⚠️ **Breaking Changes**
- Installer has been changed from `InnoSetup` (`.exe`) to `WiX Toolset` (`.msi`) in order to support deep signing of the binaries.

**ATTENTION: The MSI installer (preview) published on 24.05.2024 had an error in the uninstall routine. If you have problems with the uninstallation or the current update, first reinstall the version [2024.4.21.0 (direct download of the setup](https://github.com/BornToBeRoot/NETworkManager/releases/download/2024.4.21.0/NETworkManager_2024.4.21.0_Setup.exe). You can then run the MSI installation programme to update to the latest version.**

🚀 **New Feature**
- The binaries and the installer (`.msi`) are now signed with a free code signing certificate from [SignPath Foundation](https://signpath.org/). Thanks to [SignPath.io](https://signpath.io/) for providing free code signing for open source projects.

---

| File | Checksum [SHA256]
|---|---|
|[`NETworkManager_2024.5.27.0_Setup.msi`](https://github.com/BornToBeRoot/NETworkManager/releases/download/2024.5.27.0/NETworkManager_2024.5.27.0_Setup.msi)| `BE6A466ACD8912B21702A8D07CC45D7DEF7D177A6BFC3BDE2AB97E1DBE8E9CEA` |
|[`NETworkManager_2024.5.27.0_Portable.zip`](https://github.com/BornToBeRoot/NETworkManager/releases/download/2024.5.27.0/NETworkManager_2024.5.27.0_Portable.zip)| `3B5AE229CF692AA43B2F35B3FF9595E48E0D821B9CF8015E6AB3B1DA91B59B36` |
|[`NETworkManager_2024.5.27.0_Archive.zip`](https://github.com/BornToBeRoot/NETworkManager/releases/download/2024.5.27.0/NETworkManager_2024.5.27.0_Archive.zip)| `775AAF2BEC7434D458F9A4208B803D7E26F447EF2DEDD977030E6FA16F501BE7` |

**System requirements**
- Windows 10 / Server x64 (1809 or later)
- [.NET Desktop Runtime 8.0 (LTS) - x64](https://dotnet.microsoft.com/en-us/download/dotnet/8.0/runtime)

<hr /><em>This discussion was created from the release <a href='https://github.com/BornToBeRoot/NETworkManager/releases/tag/2024.5.27.0'>2024.5.27.0</a>.</em>

**Comment by gamefreak83** [comment]
I have a Problem with the msi package. I can't install the latest release 2024.5.27.0 on my windows 11 pro machine. Every time i get the error massage "There is a problem with this Windows Installer package. A program required for this install to complete could not be run. Contact your support personnel or package vendor.

I use Windows 11 23H2 Build 22631.3593. On my Windows 10 machine the msi package works fine.

**Reply by BornToBeRoot** [reply]
Hi @gamefreak83,

try to install the old InnoSetup first (https://github.com/BornToBeRoot/NETworkManager/releases/download/2024.4.21.0/NETworkManager_2024.4.21.0_Setup.exe) and then run the MSI installer again.

Please also verify where the program is installed. By default this should be `C:\Program Files\NETworkManager`

If this doesn't work, please continue here: https://github.com/BornToBeRoot/NETworkManager/issues/2732
